pkg/shares/non_interactive_defaults.go

```diff
@@ -83,10 +83,31 @@ func roundUpBy(cursor, v int) int {
 	}
 }
 
-// MinSquareSize returns the minimum square size that can contain shareCount
-// number of shares.
-func MinSquareSize(shareCount int) int {
-	return RoundUpPowerOfTwo(int(math.Ceil(math.Sqrt(float64(shareCount)))))
+// SubTreeWidth determines how many shares should be included when creating
+// subtree. This is used for determining which subtree roots are used in the
+// share commitment for PFBs, and therefore also the padding between blobs. The
+// reasoning behind this algorithm is discussed in depth in ADR013.
+func SubTreeWidth(shareCount int) int {
+	// per ADR013, we use a predetermined threshold to determine width of sub
+	// trees used to create share commitments
+	s := (shareCount / appconsts.SubtreeRootHeightThreshold)
+
+	// round up if the width is not an exact multiple of the threshold
+	if shareCount%appconsts.SubtreeRootHeightThreshold != 0 {
+		s++
+	}
+
+	// use a power of two equal to or larger than the multiple of the subtree
+	// root threshold
+	s = RoundUpPowerOfTwo(s)
+
+	// using a value below the min square size is wasteful, see ADR013 for more
+	// details
+	if s < appconsts.DefaultMinSquareSize {
+		return appconsts.DefaultMinSquareSize
+	}
```

Comment:
I thought we were removing the notion of a `MinSquareSize`?

Reply:
since ADR009, we've had two minsquaresizes, the above is a constant depicting the smallest viable ODS, where as the ADR009 min square size is a var that changes pers the blob for the minimum square size that the blob can fit in

Reply:
But isn't this always 1x1? When would it not be 1x1?

Reply:
It might simply be a remnant of malleation, where we might have wanted to limit the number of possible square sizes to sign over.